### PR TITLE
[R4ML-160] fix r4ml.fs.mode()

### PR DESCRIPTION
**Pull request by bdwyer2**
- Fixed file system detection bug
- Changed logo per Alok

### CODE REVIEW

**Comment by bdwyer2** [comment]
Maybe we should use return `hdfs` instead of `cluster`. What do you think @aloknsingh?

File: R4ML/R/util.io.R [hunk 1]
```diff
@@ -133,28 +133,16 @@ create.r4ml.fs <- function() {
...
   }
-  
-  #user input or env has local[*] kind of settings
-  if (substr(SparkR::sparkR.conf()$spark.master, 1, 5) == "local") {
-    return("local")
-  }
-  
-  # if the input is not one of the above it is most liklely some other kind of spark cluster
-  # TODO discuss, if it has to be local?
-  r4ml.warn(logSource, "Unable to determine file system. Defaulting to cluster mode.")
```
Comment (aloknsingh) [inline R4ML/R/util.io.R:156]: we need these lines to make sure that fs.defaults doesn't return wrong.

**Comment by aloknsingh** [comment]
@bdwyer2 can you pl confirm that jira use case is taken care as the test i.e in the cluster set the master to local but files are created and run on the cluster

**Comment by bdwyer2** [comment]
@aloknsingh it does fix the issue in the JIRA.

In regards to my previous message what do you think about using `hdfs` instead of `cluster`?

**Review by aloknsingh — APPROVED** [review]

**Comment by bdwyer2** [comment]
Jenkins, retest this please.

**Pull request closed without merging**